### PR TITLE
199 add section

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/components/TextEditorPopup.tsx [hunk 1]
```diff
@@ -122,6 +143,7 @@ const TextEditorPopup = (props: DialogProps) => {
             color="primary"
             variant="contained"
             onClick={handleCompleteClick}
+            sx={{ height: '2rem' }}
```
Comment (avirankatz) [inline frontend/src/components/TextEditorPopup.tsx:146]: - [ ] Remove this line

File: frontend/src/components/TextEditorPopup.tsx [hunk 2]
```diff
@@ -78,13 +105,7 @@ const TextEditorPopup = (props: DialogProps) => {
...
         <Stack direction="row" alignItems="center" justifyContent="space-between">
           <Stack direction="row" color={textSecondaryColor}>
-            <Typography variant="h3">{parentSection}</Typography>
-            <Typography>
-              <ArrowLogo />
-            </Typography>
-            <Typography variant="h3" color="white">
-              {newVersionName}
-            </Typography>
+            {loadTitle()}
```
Comment (avirankatz) [inline frontend/src/components/TextEditorPopup.tsx:108]:
```suggestion
            <Typography variant="h3" color={newVersionName ? undefined : 'text.primary'}>
              {parentSection}
            </Typography>
            {newVersionName && (
              <>
                <ArrowLogo />
                <Typography variant="h3" color="white">
                  {newVersionName}
                </Typography>
              </>
            )}
```

File: frontend/src/components/TextEditorPopup.tsx [hunk 3]
```diff
@@ -50,12 +52,37 @@ const TextEditorPopup = (props: DialogProps) => {
...
+        <Typography>
+          <ArrowLogo />
+        </Typography>
+        <Typography variant="h3" color="white">
+          {newVersionName}
+        </Typography>
+      </>
+    );
+  }
+
```
Comment (avirankatz) [inline frontend/src/components/TextEditorPopup.tsx:85]:
```suggestion
```

File: frontend/src/pages/Agreement.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+div#devider:hover {
```
Comment (avirankatz) [inline frontend/src/pages/Agreement.css:1]: - [ ] typo: devider > divider

File: frontend/src/pages/Agreement.tsx [hunk 5]
```diff
@@ -116,13 +127,49 @@ const Agreement: FC = () => {
...
+                            display: 'flex',
+                            alignItems: 'center',
+                          }}
+                        >
+                          <img src={PlusIcon} height="10px" width="10px" />
+                        </IconButton>
+                      </Divider>
+                      <TextEditorPopup
+                        isOpen={isTextPopupOpen}
+                        parentSection={`${t(StringBank.NEW_SECTION)}`}
```
Comment (avirankatz) [inline frontend/src/pages/Agreement.tsx:155]:
```suggestion
                        parentSection={t(StringBank.NEW_SECTION)}
```

File: frontend/src/pages/Agreement.tsx [hunk 6]
```diff
@@ -116,13 +127,49 @@ const Agreement: FC = () => {
...
-                      key={section.id}
-                      section={section}
-                      onClick={() => navigate(`section/${section.id}`)}
-                    />
+                    <div key={section.id}>
+                      <SectionCard
+                        section={section}
+                        onClick={() => navigate(`section/${section.id}`)}
+                      />
+                      <Divider id="devider" textAlign="center" variant="fullWidth">
```
Comment (avirankatz) [inline frontend/src/pages/Agreement.tsx:137]: - [ ] Use class instead of id (since there are many dividers)
- [ ] typo: devider -> divider

**Review by avirankatz — CHANGES_REQUESTED** [review]
Nice one. Please see comments/suggestions

File: frontend/src/components/TextEditorPopup.tsx (hunk 1 above)
Comment (yishain11) [reply frontend/src/components/TextEditorPopup.tsx:146]: removed, but it was added to change the button to closer resamble the zeplin one

File: frontend/src/components/TextEditorPopup.tsx (hunk 2 above)
Comment (yishain11) [reply frontend/src/components/TextEditorPopup.tsx:108]: changed

File: frontend/src/components/TextEditorPopup.tsx (hunk 3 above)
Comment (yishain11) [reply frontend/src/components/TextEditorPopup.tsx:85]: removed

File: frontend/src/pages/Agreement.css (hunk 4 above)
Comment (yishain11) [reply frontend/src/pages/Agreement.css:1]: fixed

File: frontend/src/pages/Agreement.tsx (hunk 5 above)
Comment (yishain11) [reply frontend/src/pages/Agreement.tsx:155]: changed

File: frontend/src/pages/Agreement.tsx (hunk 6 above)
Comment (yishain11) [reply frontend/src/pages/Agreement.tsx:137]: fixed

**Review by avirankatz — APPROVED** [review]

**Pull request merged**